src/ess/estia/corrections.py

```diff
@@ -29,7 +29,7 @@ def add_coords_masks_and_apply_corrections(
     bdlim: BeamDivergenceLimits,
     wbins: WavelengthBins,
     proton_current: ProtonCurrent[RunType],
-    graph: CoordTransformationGraph,
+    graph: CoordTransformationGraph[RunType],
 ) -> ReducibleData[RunType]:
```

Comment:
Not for this PR, but this reminds me that `ReducibleData` should probably be renamed to `CorrectedDetector`? (according to the new naming scheme)